server/core/src/main/java/dev/slimevr/VRServer.kt

```diff
@@ -100,6 +102,8 @@ class VRServer @JvmOverloads constructor(
 
 	init {
 		// UwU
+		instance = this
```

Comment:
what was the reason this was moved to the start instead of waiting for everything to initialize and then let instance be?

Reply:
This was for referencing vrserver as latestart or something.

Reply:
fixed